### ISSUE TITLE
Using Tailwind CSS with CRA

### ISSUE THREAD

**Issue by agneym** [tailwindcss/setup-examples#46]
For a recent project, I created a [fork of React Scripts ](https://github.com/agneym/create-react-app/tree/master/packages/react-scripts) and fetches from a [tailwind css template](https://github.com/agneym/cra-template-tailwind) pushed to npm.

It can be used with:

```
npm init react-app@next folder-name --scripts-version=@agney/rs-tailwind --template=tailwind
```
_Requires @next for now, since some fixes for template are not on major release_

I was wondering if the repositories could be moved here for better maintanence and visiblity.

**Comment by kaisheng1** [comment]
Hi, this is a really helpful template so far I can find. However, can you share how you build this template, cause I'm wondering if I can do the same thing.

**Comment by kaisheng1** [comment]
Found a brilliant template here: https://github.com/GeoffSelby/cra-template-tailwindcss. Check this repo if you would. I still want to know how you built your own template though.

**Comment by agneym** [comment]
@kaisheng1  The template you linked to uses a package named `craco` to customise the CRA from the outside. 
Mine uses a custom package that is self made: [create-react-app-tailwind](https://github.com/agneym/create-react-app-tailwind) that is forked from `create-react-app` which is customised to include tailwind and postcss.

[Auth0 has a nice tutorial](https://auth0.com/blog/how-to-configure-create-react-app/) on how to customise one on your own.

**Comment by kaisheng1** [comment]
Yeah, I was just wandering around and find the best way to do stuff. Would be really cool to make my own custom package, if it's necessary.
